### ISSUE TITLE
How to use the unlabeled data during training?

### ISSUE THREAD

**Issue by aldipiroli** [rheinzler/PointCloudDeNoising#25]
Hello @rheinzler,
I am trying to reproduce your results I have some doubts about your training and testing data. 

In the training set we have  the weather chamber data with labels [100: clear, 101: rain, 102: fog] and the unlabeled data from the road scenes [0: unlabeled].
During training, how do you treat the unlabeled data? Do your remap them to the clear class [100] or you ignore them during the loss calculation?

Similarly, in the evaluation section you mention that: 

> "Note, all evaluations are based on the test data set from experiment 2, which contains autolabeled annotations and road data without fog, rain or augmentation."

However the classes taken into consideration during the evaluation are only [100: clear, 101: rain, 102: fog]. 
Did you only evaluate on the weather chamber data or did you remap the unlabeled to the clear class?
